test/functional/rpc_psbt.py

```diff
@@ -213,31 +220,31 @@ def run_test(self):
         # replaceable arg
         block_height = self.nodes[0].getblockcount()
         unspent = self.nodes[0].listunspent()[0]
-        psbtx_info = self.nodes[0].walletcreatefundedpsbt([{"txid":unspent["txid"], "vout":unspent["vout"]}], [{self.nodes[2].getnewaddress():unspent["amount"]+1}], block_height+2, {"replaceable": False}, False)
+        psbtx_info = self.nodes[0].walletcreatefundedpsbt([{"txid":unspent["txid"], "vout":unspent["vout"]}], [{self.nodes[2].getnewaddress():unspent["amount"]+1}], block_height+2, {"replaceable": False, "add_inputs": True}, False)
```

Comment:
Can this be tested for the `"add_inputs": False` case as well?

Reply:
I don't think that's very useful, because the goal of the test is to "Make sure both pre-included and funded inputs have the correct sequence numbers", and `"add_inputs": False` would just test a subset of what we already check here.